kong/plugins/zipkin/schema.lua

```diff
@@ -65,6 +65,7 @@ return {
           { connect_timeout = typedefs.timeout { default = 2000 } },
           { send_timeout = typedefs.timeout { default = 5000 } },
           { read_timeout = typedefs.timeout { default = 5000 } },
+          { response_header_for_traceid = { type = "string", default = nil }},
```

Comment:
Will it be better to add an `http_` prefix? It only works in the HTTP mode.

Reply:
field renamed to `http_response_header_for_traceid`